nixos/modules/services/web-servers/minio.nix

```diff
@@ -29,6 +29,38 @@ in
       description = "The config directory, for the access keys and other settings.";
     };
 
+    accessKey = mkOption {
+      default = "";
+      type = types.str;
+      description = ''
+        Specify the Secret key of 8 to 40 characters in length that clients use to access the server.
+        If not set the generated key from <literal>configDir</literal> is used.
+      '';
+    };
+
+    secretKey = mkOption {
+      default = "";
+      type = types.str;
+      description = ''
+        Access key of 5 to 20 characters in length that clients use to access the server.
+        If not set the generated key from <literal>configDir</literal> is used.
```

Comment:
@Mic92 @0xABAB I tried to rephrase the description to make it more where the generated keys come from.